docs/Windows.md

```diff
@@ -91,10 +92,49 @@ Uberzugpp should work on WSL. You might have to configure it in a special way.
 The title of that discussion says it all. This is a cursed method of using kitty in a graphical WSL
 session.
 
-#### Just use `:MoltenImagePopup`
-
-The MoltenImagePopup is the officially supported way to view images on windows. It's a far worse
-experience, but you can configure `vim.g.molten_auto_image_popup = true` to at least see images
-automatically when your code produces them.
-
-This is the only method known to work without using WSL.
+#### `:MoltenImagePopup`
+
+The first officially supported way to view images on Windows. It's a far worse experience compared
+to the other methods, but you can configure `vim.g.molten_auto_image_popup = true` to at least see
+images automatically when your code produces them. It's also very simple, and doesn't have any
+dependencies.
+
+#### Wezterm (via Wezterm.nvim)
+
+The second officially supported way to render images without the use of WSL on Windows is to use
+Wezterm as your terminal emulator and setting `vim.g.molten_image_provider = "wezterm"`. This is
+a bit of a workaround, but it's the only other way to get images to render in the terminal without
+needing an external pop-up window like used with `:MoltenImagePopup`. You can find the instructions
+for downloading and setting up Wezterm [here](https://wezfurlong.org/wezterm/install/windows.html).
+
+![](wezterm.gif)
```

Comment:
Can you upload this gif to github and then insert it like that so that it's not hosted in the repo. Just so people don't have to download the gif when they clone the repo.